src/plugins/media-analysis/media-analysis-editor.js

```diff
@@ -165,6 +175,51 @@ function MediaAnalysisEditor({ content, onContentChanged }) {
     changeContent({ volumePresets: newVolumePresets });
   };
 
+  const handleFileDrop = async fs => {
```

Comment:
It's cool that drag and drop is supported, but it's not visible to the user 🤔 
Should we maybe have a tooltip over the menu option saying that "this can be done via drag and drop too"?

Reply:
True. I added a small icon with tooltip at the lower right corner of the box

Reply:
Nice idea!